lang/en/subcourse.php

```diff
@@ -71,6 +71,8 @@
 $string['settings:coursepageprintgrade_desc'] = 'Display grade from referenced course on course page.';
 $string['settings:coursepageprintprogress'] = 'Progress on course page';
 $string['settings:coursepageprintprogress_desc'] = 'Display progress from referenced course on course page.';
+$string['settings:limittoenrolled'] = 'Limit to courses the trainer is enrolled to';
+$string['settings:limittoenrolled_desc'] = 'This shows only courses the trainer is enrolled to. Improves performance on big instances.';
 $string['subcourse:addinstance'] = 'Add a new subcourse';
```

Comment:
"trainer" is not a term used in the Moodle English language pack. If it is confirmed that this settings actually works (see other comments), then I would prefer something like: "Limit the options in the field 'Fetch grades from' to the list of courses the user is enrolled to" or so.